styles.css

```
@@ -643,6 +651,190 @@ If your plugin does not need CSS, delete this file.
+		width 140ms ease,
+		height 140ms ease;
+}
+
+.agent-client-turn-navigator-label {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)